packages/odyssey-icons/package.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "@okta/odyssey-icons",
+  "version": "0.3.0",
+  "description": "The home for Odyssey's supported icons.",
+  "homepage": "https://github.com/okta/odyssey",
+  "license": "Apache-2.0",
+  "author": "Okta, Inc.",
+  "files": [
+    "src"
+  ],
+  "scripts": {
+    "svgo": "svgo --config='.svgo.yml' -f src"
```

Hoping to get a few more details as to the expected behavior here:
- Looks like this is going to convert all of our source files in place instead of outputting them to a `dist` directory. Do we want to modify our actual source files? I'd recommend using the `-o dist` command to ensure these go out to a separate folder to avoid rewriting assets.
- This is also very similar to a build-time command, as we're setting up our files to be consumed by other packages. If we rename this to named script to `build` - we can then call move toward leveraging `lerna`'s  [`run`](https://github.com/lerna/lerna/tree/master/commands/run#usage) command to build all of our packages on `install`.

```suggestion
    "build": "svgo --config='.svgo.yml' -f src -o dist"
```

---

This is a utility command for prepping SVGs, not a build command for source materials within the package. The canonical icons are likely always going to be sourced from Figma, not code - this is solely for ensuring they're optimized against the same configuration and is idempotent when run against the existing files.

---

I've changed the path to `dist` to make this less confusing.